_data/security.yml

```diff
@@ -82,6 +89,7 @@ websites:
       img: gogetssl.png
       tfa: No
       twitter: gogetssl
+      facebook: gogetssl
```

Comment:
Why is GoGetSSL changed in this PR?

Reply:
mainly because when I broke one large PR into 13-15 smaller PRs, I grouped changes that were close to each other in the same file. Since this is an issue, I'll remove it and make the gogetssl change a different PR.